diff --git a/node_modules/@agoric/smart-wallet/src/marshal-contexts.js b/node_modules/@agoric/smart-wallet/src/marshal-contexts.js
index de5b3b9..b7594b6 100644
--- a/node_modules/@agoric/smart-wallet/src/marshal-contexts.js
+++ b/node_modules/@agoric/smart-wallet/src/marshal-contexts.js
@@ -286,7 +286,7 @@ export const makeImportContext = (makePresence = defaultMakePresence) => {
      * @param {string} iface
      */
     fromBoard: (slot, iface) => {
-      isDefaultBoardId(slot) || Fail`bad board slot ${q(slot)}`;
+      isDefaultBoardId(slot) || slot === null || Fail`bad board slot ${q(slot)}`;
       return provideVal(boardObjects, slot, iface);
     },
 
